docs/ui/components/Authentication/index.ts

```diff
@@ -29,4 +28,5 @@ export const ASSETS = {
   twitch: ASSETS_PATH + 'twitch.png',
   twitter: ASSETS_PATH + 'twitter.png',
   uber: ASSETS_PATH + 'uber.png',
+  calendly: ASSETS_PATH + 'calendly.svg',
```

Comment:
Same as above. On this page, we follow alphabetical order so let' stick to that. Please remove this line from here and add it before "Cognito".